model/channel.go

```diff
@@ -27,6 +27,7 @@ type Channel struct {
 	Description   string `json:"description"`
 	LastPostAt    int64  `json:"last_post_at"`
 	TotalMsgCount int64  `json:"total_msg_count"`
+	ExtraUpdateAt int64  `json:"extra_update_at"`
```

Comment:
We cannot use channel.UpdateAt vs adding a new col just for the etag stuff?  why not just increment Channel.UpdateAt?